consoles/sshVirtsh.pm

```diff
@@ -515,6 +515,11 @@ sub add_disk ($self, $args) {
         $file = $self->_create_disk($args, $vmware_openqa_datastore, $file, $name, $basedir);
     }
     else {
+        # When the s390-kvm boots copies HDD_. cleanup old qcow image before proceed
+        if ($args->{file} && $args->{cdrom} || $args->{backingfile}) {
```

Comment:
why the cdrom when above you say HDD?

Reply:
Definetely i need to make this clear. I am getting the understanding bit by bit as I go. 
A quick answer is because of `($cdrom || $backingfile)` in `_copy_image_to_vm_host` but... Consider everything wrong for now until I am sure that the change actually removes the expected files from the expected destination without defects. I will mark the PR as draft for now